public/css/icinga/login.less

```diff
@@ -103,6 +103,11 @@
     }
   }
 
+  input:focus,
```

Comment:
Please remove this. For the general input types this is already the case. Only the submit button requires this.